app/View/Elements/Events/View/value_field.ctp

```diff
@@ -2,10 +2,16 @@
   $sigDisplay = $object['value'];
   if ('attachment' == $object['type'] || 'malware-sample' == $object['type'] ) {
     if ($object['type'] == 'attachment' && isset($object['image'])) {
-      $extension = explode('.', $object['value']);
-      $extension = end($extension);
-      $uri = 'data:image/' . strtolower(h($extension)) . ';base64,' . h($object['image']);
-      echo '<img class="screenshot screenshot-collapsed useCursorPointer" src="' . $uri . '" title="' . h($object['value']) . '" />';
+        if (extension_loaded('gd')) {
+            $img = '<it class="fa fa-spin fa-spinner" style="font-size: large; left: 50%; top: 50%;"></it>';
+            $img .= '<img class="screenshot screenshot-collapsed useCursorPointer img-rounded" src="' . $baseurl . '/attributes/viewPicture/' . h($object['id']) . '/1' . '" title="' . h($object['value']) . '" onload="$(this).parent().find(\'.fa-spinner\').remove();"/>';
```

Comment:
This looks great, however, keep in mind that the image is still loaded, b64 encoded and passed to the view - it just isn't transferred to the user. Perhaps also change the bahaviour on the controller side (EventsController->view() / EventsController->__viewUI() / EventsController->viewEventAttributes() - or whatever it's called).

Reply:
Sure! Especially that this PR aims to improve the perf...
will have a look.